src/org/usfirst/frc/team2706/robot/RobotMap.java

```diff
@@ -64,7 +64,7 @@
     private static final boolean[] INVERT_JOYSTICK_X_VALS = {true, true, true};
     public static final boolean INVERT_JOYSTICK_X = getConstant("INVERT_JOYSTICK_X");
 
-    private static final boolean[] INVERT_JOYSTICK_Y_VALS = {false, false, false};
+    private static final boolean[] INVERT_JOYSTICK_Y_VALS = {true, false, false};
```

Comment:
This is going to conflict with the  2017 robot, so one of the robots should become the practice robot, and one is competition.